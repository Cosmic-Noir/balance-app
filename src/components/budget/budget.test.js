import React from "react";
import ReactDom from "react-dom";
import renderer from "react-test-renderer";
import Budget from "./budget";

// Unclear how to write these tests with a DOM element registering as null

describe(`Budget Component`, () => {
  it(`Renders without crashing`, () => {
    const div = document.createElement("div");
    ReactDom.render(<Budget testingVal="Feb 2020" />, div);
    ReactDom.unmountComponentAtNode(div);
  });

  it(`Renders the UI as expected`, () => {
    const tree = renderer.create(<Budget testingVal="Feb 2020" />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
