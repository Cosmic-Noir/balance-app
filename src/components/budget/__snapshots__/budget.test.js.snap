// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Budget Component Renders the UI as expected 1`] = `[Function]`;
